File: web dev/Java Script/basic/Array.js
arr =[112,12,14,41];
console.log(arr); // [ 112, 12, 14, 41 ]

n =arr.length;
console.log(n); //4
arr.push(4); // add at last

for(let i =0; i<arr.length;i++){
    process.stdout.write(arr[i]+" ");
} //112 12 14 41 4
console.log();

arr.pop(); // remove last element
console.log(arr);

arr.unshift(2); // add at 0th place
console.log(arr);

arr.shift(); // remove first element
console.log(arr);

File: C language/structue/array.c
#include<stdio.h>
void main(){
    int n;
    printf("enter n:- ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter element:- ");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    int su=0;
    for(int i=0;i<n;i++){
    su+=arr[i];
    }
    printf("%d",su);
}